.github/workflows/nightly.yml

```diff
@@ -0,0 +1,47 @@
+name: Nightly Build
+
+on:
+  schedule:
+    - cron: '0 2 * * *' # run at 2 AM UTC
+  workflow_dispatch:
+
+jobs:
+  nighlty:
+    name: Nightly build job
+
+    runs-on: ${{ matrix.os }}
+
+    continue-on-error: true
```

Comment:
Now it will also continue on error for node-14, which is what we don't want right?

Reply:
Correct. 
- Long term we have to come up with a solution that fits our needs
- Short term: are you able to push a small change to improve on this? And if possible, can you also include the matrix change? (your previous comment)